lib/HiddenStackError.js

```diff
@@ -0,0 +1,11 @@
+/*
+	MIT License http://www.opensource.org/licenses/mit-license.php
+*/
+"use strict";
+
+module.exports = class HiddenStackError extends Error {
```

Comment:
Inherit from WebpackError like the other Error classes.

Maybe you can make the class more meaningful by naming it like `HarmonyLinkingError`?